cluster.go

```diff
@@ -1597,3 +1597,41 @@ func diffPeers(peers1, peers2 []peer.ID) (added, removed []peer.ID) {
 	}
 	return
 }
+
+// RepoGC performs garbage collection sweep on all peers' repo.
+func (c *Cluster) RepoGC(ctx context.Context) error {
```

Comment:
Can we get an endpoint that allows for individual gc? We can then trigger gc on particular peers based metrics coming in

Reply:
> Can we get an endpoint that allows for individual gc? We can then trigger gc on particular peers based metrics coming in

You mean, a public endpoint? Internally this should be no problem (talking directly to the IPFSConnetor).